packages/Link/Link.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { forwardRef } from 'react'
+import PropTypes from 'prop-types'
+import { css } from '@emotion/core'
+import { parkGreen } from '@pm-kit/colours'
+
+const linkStyle = css`
+  color: ${parkGreen};
+  &::hover {
+    color: ${parkGreen};
+  }
+`
+
+const Link = ({ children, forwardedRef, link, target, ...rest }) => {
+  return (
+    <a css={linkStyle} href={link} target={target} rel="noopener noreferrer">
```

Comment:
This shouldn't be the default rel value. This means we don't pass referring data to the new page, and if this is hardcoded, will have a negative SEO impact. This value should only be there if the target is blank. So either the developer using this component puts it in themself, or we can also offer it as a default in the component if they put in a `target="_blank"`

"Definitely do not use the rel=”noreferrer” attribute on internal links, it can mess up with your Google analytics reports."
https://www.reliablesoft.net/noreferrer-noopener/